lib/hawkular/alerts/alerts_api.rb

```diff
@@ -20,6 +20,13 @@ def initialize(entrypoint = 'http://localhost:8080/hawkular/alerts', credentials
       super(entrypoint, credentials, options)
     end
 
+    # Return version and status information for the used version of Hawkular-Alerting
+    # @return [Hash{String=>String}]
+    #         ('Implementation-Version', 'Built-From-Git-SHA1', 'status')
+    def fetch_version_and_status
```

Comment:
My opinion is that change should go in  a following PR where we can review and upgrade version later.